build.gradle

```diff
@@ -9,6 +9,7 @@ buildscript {
     }
     dependencies {
         classpath 'com.android.tools.build:gradle:3.1.3'
+        classpath 'com.github.jengelman.gradle.plugins:shadow:2.0.4'
```

Comment:
Have we done an OSS component review for this?

Reply:
Nope, Thanks for pointing out. Will do that. For test only dependencies, that actually just means registering the component in Component Governance.